docs/intro/building.rst

```diff
@@ -63,7 +63,8 @@ The code can be built as
 .. code-block:: bash
 
     mkdir build && cd build
-    cmake -DCMAKE_PREFIX_PATH="<install_dir>/kokkos;<install_dir>/kokkos_fft" ..
+    cmake -DCMAKE_CXX_COMPILER=icpx \
+          -DCMAKE_PREFIX_PATH="<install_dir>/kokkos;<install_dir>/kokkosFFT" ..
```

Comment:
This line would become

```
    -DKMAKE_PREFIX_PATH=path/to/kokkos;path/to/kokkos-fft`
```

Alternatively, you can use the syntax:

```
    -DKokkos_ROOT=path/to/kokkos
    -DKokkosFFT_ROOT=path/to/kokkos-fft
```

The latest version of the cheat sheet tends to use this way.

Reply:
In the future release, we may use 

```
    -DKokkos_ROOT=path/to/kokkos
    -DKokkosFFT_ROOT=path/to/kokkos-fft
```


For the moment, we use the `-DKMAKE_PREFIX_PATH` version in CI, so I would keep in the docs as well

Reply:
Ok.